pkg/node/fsm/action.go

```diff
@@ -74,3 +75,25 @@ func (a *ActionsImpl) SendBlock(block *proto.Block) {
 	a.logger.Debug("Network message sent to peers", logging.Type(msg), slog.Int("count", cnt),
 		slog.Any("blockID", block.BlockID()))
 }
+
+func (a *ActionsImpl) SendEndorseBlock(endorse *proto.EndorseBlock) {
+	bts, err := endorse.Marshal()
+	if err != nil {
+		a.logger.Error("Failed to marshal endorse block", logging.Error(err))
+		return
+	}
+
+	msg := &proto.EndorseBlockMessage{
```

Comment:
Does this message have `Stringer`interface for user-readable representation in debug logging mode, for example?